props/saber_fett263_buttons.h

```diff
@@ -1573,6 +1573,30 @@ SaberFett263Buttons() : PropBase() {}
     }	  
   }
 
+  bool CheckInteractivePreon() {
```

Comment:
I think this function (and the one below) may be generically useful, so you should put them in prop_base.h. They will not take any space if not called.

Reply:
moved